main.cc

```diff
@@ -715,6 +715,7 @@ update()
     auto want_relative_mouse_mode = SDL_bool(window_has_focus() && !current_game_state->is_ui_state());
     if (SDL_GetRelativeMouseMode() != want_relative_mouse_mode) {
```

Comment:
Why here and not in ... set_game_state or whatever the deferred bit is?

Reply:
Gain/loss of focus